.rubocop.yml

```diff
@@ -24,6 +24,12 @@ Layout/EmptyLineAfterGuardClause:
 Layout/FirstParameterIndentation:
   Enabled: false
 
+Layout/EmptyLinesAroundAttributeAccessor:
+  Enabled: true
```

Comment:
Don't all the rules come enabled? Do we need to explicitly enable these?

Reply:
I would think so but I got this message for the cops that I added to the file: `The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.`

Reply:
Apparently so then 😊

Reply:
ah, I see you can add something to the file indicating you want to opt-in to all new cops.